interface_tester/collector.py

```diff
@@ -66,7 +66,7 @@ class _InterfacesDotYamlSpec(TypedDict):
 
     providers: List[_CharmTestConfig]
     requirers: List[_CharmTestConfig]
-    owners: List[str]
+    maintainers: str
```

Comment:
Should this be a `List[str]` as it's maintainers (plural)?

Reply:
The value will be a single GitHub team ID, not a list of IDs, as in "a team of maintainers". Shall we make it "maintainer"?

Reply:
Yes, in that case, let's please make it `maintainer` singular.